### DISCUSSION TITLE
Installing issue: CondaEnvException

### DISCUSSION THREAD

**Discussion by KQEFEM** [pypsa-meets-earth/pypsa-earth#702] [Q&A]
Hi All, 
I am trying to install the package using 

mamba env create-f envs/environment.yaml 

but I keep getting the error in the image. I have git installed and it appears to work without errors but I still get this below. 

Any advice would be welcome :) 

![image](https://user-images.githubusercontent.com/34337935/235705278-c26880b2-4d0e-4cc9-8e76-eca242dddea6.png)

**Comment by ekatef** [comment]
Hey @KieranSQ and welcome!

Thank you for reporting the installation issue. It looks like you are facing troubles when installing dependencies from github. According to this [SO discussion](https://stackoverflow.com/questions/29565779/cannot-find-command-git-windows), installation of git and/or setting a PATH variable for git can help
